utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/SchemaWithMetadata.scala

```diff
@@ -11,7 +11,7 @@ import pl.touk.nussknacker.engine.kafka.SchemaRegistryClientKafkaConfig
  * It is lightened version of Confluent's SchemaMetadata. We don't want to use their class, because our SchemaRegistryClient
  * is not coupled with concrete schema registry implementation.
  */
-case class SchemaWithMetadata private(schema: ParsedSchema, id: Int)
+case class SchemaWithMetadata private(schema: ParsedSchema, id: SchemaId)
```

Comment:
maybe we should move this class to `confluent` package?

Reply:
why to confluent package ? it should not be there

Reply:
I moved Confluent based code to `ConfluentUtils`

Reply:
@dswiecki it is method that converts Confluent class to ours so it should be in the confluent package. Why do you think that it shouldn't?